cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/connect/class-api.php

```diff
@@ -230,6 +230,7 @@ function ( $item ) use ( $transformation_index ) {
 	public function cloudinary_url( $public_id, $args = array(), $size = array(), $clean = false ) {
 		$defaults = array(
 			'resource_type' => 'image',
+			'version'       => '',
```

Comment:
@pereirinha - can you change the default for the version to v1?
```suggestion
			'version'       => '',
```